runtime/sema/type.go

```diff
@@ -4289,6 +4289,21 @@ var authAccountAddPublicKeyFunctionType = &FunctionType{
 	),
 }
 
+var authAccountAddAccountKeyFunctionType = &FunctionType{
```

Comment:
IMO this should just replace the `addPublicKey` function (and maybe even keep the same name).

Reply:
Yep, in general this will get refactored in a follow up PR to use the new `KeyList` type, so we can keep this for now